test/test_helper.rb

```diff
@@ -14,6 +15,18 @@ class ActiveSupport::TestCase
 class ActionDispatch::IntegrationTest
   include Capybara::DSL
 
+  def make_admin
+    User.create(username: "admin", password: "pw", role: 1)
+  end
+
+  def logged_in_user
+    User.create(username: "user", password: "password")
+    visit '/login'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.